conference.js

```diff
@@ -1799,6 +1803,7 @@ export default {
      */
     hangup (requestFeedback = false) {
         APP.UI.hideRingOverLay();
+        APP.remoteControl.receiver.enable(false);
```

Comment:
For me it makes more sense to have some event for hangup/conference ended and handle it in the remoteControl.receiver directly